examples/misc/lib/effective_dart/docs_good.dart

```diff
@@ -64,19 +68,17 @@ void miscDeclAnalyzedButNotTested() {
 
   <T>() {
     // #docregion third-person
-    /// Returns `true` if every element satisfies the [predicate].
-    bool all(bool predicate(T element)) => ellipsis();
```

Comment:
This is precisely the kind of function this change is about, so it's not a "good" example of using a verb-documentation.
(It's also the example people keep pointing to as why they use "Returns".)